src/Servant/Multipart.hs

```diff
@@ -366,6 +496,13 @@ instance MultipartBackend Tmp where
     type MultipartBackendOptions Tmp = TmpBackendOptions
 
     defaultBackendOptions _ = defaultTmpBackendOptions
+    -- streams the file from disk 
+    loadFile _ fp =
```

Comment:
this is phisy. Why `Tmp` backend hits file-system?

There's something wrong in the design if we need to do that. In general, on a client side `Tmp` backend would read from in-memory `ByteString`s.